### ISSUE TITLE
color wheel image locations

### ISSUE THREAD

**Issue by allanj-uaag** [w3c/low-vision-a11y-tf#39]
Section 3.1.3 – I really like the color wheel examples from 2.4.5. If you think people might be jumping around in this resource it might be worth repeating the color wheels here.
http://w3c.github.io/low-vision-a11y-tf/requirements.html#color-vision

from https://lists.w3.org/Archives/Public/public-low-vision-a11y-tf/2016Mar/0078.html

**Comment by shawna-slh** [comment]
Yea that he likes them. :-)

Summary: I suggest not repeating them.
Rationale & additional thoughts below.

I think repeating them would be distracting from the specific user needs in this section.

[side note: The point of the color wheels applies to 3.1.2 as well. Also, we might add others in 3.1 that they apply to. So if we repeated them, it would be at the top of 3.1.]

I did think about adding a pointer from the intro of 3.1. back to the color wheels (section 2.4.5) -- then looked at what other sections from 2.4 apply to 3.1 (2.4.2 and 2.4.3) -- then thought about consistency and adding to the intro of all sections of 3 pointers back to relevant sections of 2.4... and think it would be way more clutter than helpful.
